packages/angular_devkit/build_angular/test/browser/bundle-budgets_spec_large.ts

```diff
@@ -64,66 +65,82 @@ describe('Browser Builder bundle budgets', () => {
     await run.stop();
   });
 
-  it(`shows warnings for large component css when using 'anyComponentStyle' when AOT`, async () => {
-    const overrides = {
-      aot: true,
-      optimization: true,
-      budgets: [{ type: 'anyComponentStyle', maximumWarning: '1b' }],
-    };
-
-    const cssContent = `
-      .foo { color: white; padding: 1px; }
-      .buz { color: white; padding: 2px; }
-      .bar { color: white; padding: 3px; }
-    `;
+  cssExtensions.forEach(ext => {
+    it(`shows warnings for large component ${ext} when using 'anyComponentStyle' when AOT`, async () => {
+      const overrides = {
+        aot: true,
+        optimization: true,
+        budgets: [{ type: 'anyComponentStyle', maximumWarning: '1b' }],
+      };
 
-    host.writeMultipleFiles({
-      'src/app/app.component.css': cssContent,
-      'src/assets/foo.css': cssContent,
-      'src/styles.css': cssContent,
+      const cssContent = `
+        .foo { color: white; padding: 1px; }
+        .buz { color: white; padding: 2px; }
+        .bar { color: white; padding: 3px; }
+      `;
+
+      host.writeMultipleFiles({
+        [`src/app/app.component.${ext}`]: cssContent,
+        [`src/assets/foo.${ext}`]: cssContent,
+        [`src/styles.${ext}`]: cssContent,
+      });
+
+      host.replaceInFile(
+        'src/app/app.component.ts',
+        './app.component.css',
+        `./app.component.${ext}`,
+      );
+
+      const logger = new logging.Logger('');
+      const logs: string[] = [];
+      logger.subscribe(e => logs.push(e.message));
+
+      const run = await architect.scheduleTarget(targetSpec, overrides, { logger });
+      const output = await run.result;
+      expect(output.success).toBe(true);
+      expect(logs.length).toBe(2);
+      expect(logs.join()).toMatch(`WARNING.+app\.component\.${ext}`);
+      await run.stop();
     });
-
-    const logger = new logging.Logger('');
-    const logs: string[] = [];
-    logger.subscribe(e => logs.push(e.message));
-
-    const run = await architect.scheduleTarget(targetSpec, overrides, { logger });
-    const output = await run.result;
-    expect(output.success).toBe(true);
-    expect(logs.length).toBe(2);
-    expect(logs.join()).toMatch(/WARNING.+app\.component\.css/);
-    await run.stop();
   });
 
-  it(`shows error for large component css when using 'anyComponentStyle' when AOT`, async () => {
-    const overrides = {
-      aot: true,
-      optimization: true,
-      budgets: [{ type: 'anyComponentStyle', maximumError: '1b' }],
-    };
-
-    const cssContent = `
-      .foo { color: white; padding: 1px; }
-      .buz { color: white; padding: 2px; }
-      .bar { color: white; padding: 3px; }
-    `;
+  cssExtensions.forEach(ext => {
+    it(`shows error for large component ${ext} when using 'anyComponentStyle' when AOT`, async () => {
+      const overrides = {
+        aot: true,
+        optimization: true,
```

Comment:
We can add `styles: [`src/styles.${ext}`]` here so that we are sure that the budget only applies to component styles?